tests.py

```diff
@@ -226,20 +226,27 @@ def test_get_sld_top_convenience_function_is_the_same_as_PublicSuffixList_method
     def test_get_tld_returns_correct_tld_or_etld(self):
         psl = publicsuffix.PublicSuffixList()
         assert 'com' == psl.get_tld('com')
+        assert 'ck' == psl.get_tld('ck')
         assert 'kobe.jp' == psl.get_tld('city.kobe.jp')
-        assert 'kobe.jp' == psl.get_tld('kobe.jp')
-        assert 'amazonaws.com' == psl.get_tld('amazonaws.com')
+        assert 'jp' == psl.get_tld('kobe.jp')
+        assert 'com' == psl.get_tld('amazonaws.com')
         assert 'com.pg' == psl.get_tld('telinet.com.pg', wildcard=True)
-        assert 'pg' == psl.get_tld('telinet.com.pg', wildcard=False)
+        assert None == psl.get_tld('telinet.com.pg', wildcard=False)
```

Comment:
We can't use `*.pg` when we specify `wildcard=False` . Moreover, publicsuffix list doesn't contain `pg` .

Reply:
not sure i agree here.  certain companies remove the wildcards in the PSL so they can reduce the variety of output. if *.pg is in the list, which it usually is, then, when they remove the wildcard, pg will be in the list.  It doesn't remove the entry itself, just the wildcard. 

we could argue that users shouldn't use the list this way (but they do), and i don't think we want to have a situation where a user puts in a legitimate domain and gets None for the tld.  In this particular case, e.g., 'pg' is a real TLD, so it should never return None.

Reply:
Thanks to this, I now understand the semantics of `wildcard=False`. I fix the behavior on c6575d52c0d48ba7f981f2c015f31c4c90c05a73 .